large-message-connect/src/test/java/com/bakdata/kafka/AmazonS3IntegrationTest.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
We could extract this as a test-fixture into the core module

Reply:
Do you have an example for how to set it up?

Reply:
Not really one that is self-containing, but something like this should work:
1. Apply the  `java-test-fixtures` to the core module
2. Move `AmazonS3IntegrationTest` to `large-message-core/src/testFixtures/...`
3. Add `testImplementation(testFixtures(project(":large-message-core")))` to the streams and connect projects